user/message.inc

```diff
@@ -437,10 +437,13 @@ function get_uploaded_image_url($filename) {
 
     if ($result != "") {
       $j = json_decode($result, true);
-      if ($j["data"] && $j["data"]["link"])
-        return $j["data"]["link"];
-      else
+      if ($j["data"] && $j["data"]["link"]) {
+      	$iu = preg_replace("/^http:/", "https:", $j["data"]["link"]);
+        error_log($iu . " delete url: https://imgur.com/delete/" . $j["data"]["deletehash"]);
```

Comment:
Instead of logging an error, perhaps it would be best to include the url in the "changes" log that the admin can see?